src/main.py

```diff
@@ -40,4 +41,8 @@ def __init__(self, argv=None):
     if settings.value("xml_path", "") == "":
         window.mainframe.settingsWindow.show()
 
+    if (settings.value("run_game_on_startup", "false").lower() == "true"
+    and "HuntGame.exe" not in subprocess.check_output(['tasklist', '/FI', 'IMAGENAME eq HuntGame.exe']).decode()):
```

Comment:
This is not absolutely necessary and could be removed if the additional import is not wanted. Without it starting the logger would try to run the game again. That means the splash screen is shown and focus will eventually switch to the game. With this check things are just a bit nicer in case the game is already running.

Reply:
I like it!  I think I'll make a couple minor changes, to be sure the user's already set their hunt directory before it tries to launch Hunt simultaneously...just to make it more friendly for new users.
Thanks!!